### ISSUE TITLE
Stop adding spawnpoints in Level Mode

### ISSUE THREAD

**Issue by bbdoc** [Map-A-Droid/MAD#549]
Level mode should stop adding spawns to trs_spawn table, as those spawns have no value (unless you are scanning Paris, but even then, Level mode is not the right way to discover new spawns). This  increases the size of the trs_spawn table which has a very big impact on performance.

**Comment by sn0opy** [comment]
Gotcha. Not sure if it's THAT easy to implement, since it happens in different parts of MAD.

**Comment by bbdoc** [comment]
Yes, I can guess that's not that straightforwed. But I deleted 30k spawnpoints from my DB (whole Paris area) on a total of 120k and noticed a drop of my load from 5 to 3.5... so would really be useful to improve performance.

Btw, might even be useful to stop adding spawnpoints to DB also for raids and quest scans. I really only need spawnpoints for areas I'm scanning for mons. Maybe make it configurable if people still want them...

**Comment by JimBim89** [comment]
Would also be nice to have an option to stop adding them in quest mode.

**Comment by Christian-X** [comment]
I see the point, but please, make this an option if ever implemented. 
I like the idea of gathering all kind of information whenever possible... (and having new spawns to be automatically detected or refined when scanning quests is a plus)

**Comment by bbdoc** [comment]
I agree this should be optional. And I hope it can even be available for all modes, I don't see the point adding spawnpoints during quest mode either, that's a lot of rows in my trs_spawn that are really not needed and slowing down my system a lot.

**Comment by ColdShadow80** [comment]
I find that adding spawn points in "normal" quest mode is useful.
Adding spawnpoins in level mode however is not usefull and increments useless data to database

**Comment by dkmur** [comment]
I agree the amount of spawnpoints in trs_spawn has a significant impact on performance, allthough level mode could be used in seperate instance it would be great to have the option on raid/quest/level areas to de-activate adding of spawnpoints.
